.env

```diff
@@ -0,0 +1 @@
+GOOGLE_API_KEY="AIzaSyD8yprnt3VqquSST_aCqlNueOMdp_xdvAQ"
```

Comment:
Please remove this file and add it to gitignore. Instead add a `.env.example` file with an dummy key so that developers can create their own .env file from it.